packages/zoe/src/roleConversion.js

```diff
@@ -0,0 +1,20 @@
+import { assert, details } from '@agoric/assert';
+
+export const arrayToObj = (array, roleNames) => {
```

Comment:
If this is the temporary scaffolding, please say so here.

Reply:
Nope, it's not temporary.